lib/services/cloud-build-service.ts

```diff
@@ -145,13 +146,12 @@ export class CloudBuildService extends CloudService implements ICloudBuildServic
 			buildProps = await this.$nsCloudBuildPropertiesService.getiOSBuildProperties(projectSettings, buildProps, filesToUpload, iOSBuildData);
 		}
 
-		this.emitStepChanged(buildId, constants.BUILD_STEP_NAME.BUILD, constants.BUILD_STEP_PROGRESS.START);
+		this.emitStepChanged(cloudOperationId, constants.BUILD_STEP_NAME.BUILD, constants.BUILD_STEP_PROGRESS.START);
 		const buildResponse: IServerResponse = await this.$nsCloudServerBuildService.startBuild(buildProps);
 		this.$logger.trace("Build response:");
 		this.$logger.trace(buildResponse);
-		await this.waitForServerOperationToFinish(buildId, buildResponse);
-		const buildResult: IBuildServerResult = await this.getObjectFromS3File<IBuildServerResult>(buildResponse.resultUrl);
-		this.emitStepChanged(buildId, constants.BUILD_STEP_NAME.BUILD, constants.BUILD_STEP_PROGRESS.END);
+		const buildResult: ICloudOperationResult = await this.waitForServerOperationToFinish(cloudOperationId, buildResponse, { silent: false });
```

Comment:
In the previous code, we used to download the results (getObjectFromS3File), but now, we are not

Reply:
Ignore this: I was wrong